doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -239,9 +239,14 @@ nRF Desktop
   * The :ref:`nrf_desktop_factory_reset`.
     The module is used by configurations that enable :ref:`nrf_desktop_bluetooth_guide_fast_pair` to factory reset both Fast Pair and Bluetooth non-volatile data.
     The factory reset is triggered using the configuration channel.
+  * The :ref:`nrf_desktop_dfu_lock`.
+    The utility provides synchronization mechanism for accessing the DFU flash.
+    It is especially useful for application configurations that support more than one DFU method.
```

Comment:
I would remove word `especially` - the functionality is not needed at all if only one DFU transport is used.